class io.netty.util.internal.SocketUtils$1 extends java.lang.Object implements java.security.PrivilegedExceptionAction
{
final java.net.Socket val$socket;
final java.net.SocketAddress val$remoteAddress;
final int val$timeout;
void <init>(java.net.Socket, java.net.SocketAddress, int)
{
java.net.Socket v1;
java.net.SocketAddress v2;
int v3;
io.netty.util.internal.SocketUtils$1 v0;
v0 := @this: io.netty.util.internal.SocketUtils$1;
v1 := @parameter0: java.net.Socket;
v2 := @parameter1: java.net.SocketAddress;
v3 := @parameter2: int;
v0.<io.netty.util.internal.SocketUtils$1: java.net.Socket val$socket> = v1;
v0.<io.netty.util.internal.SocketUtils$1: java.net.SocketAddress val$remoteAddress> = v2;
v0.<io.netty.util.internal.SocketUtils$1: int val$timeout> = v3;
specialinvoke v0.<java.lang.Object: void <init>()>();
return;
}
public java.lang.Void run() throws java.io.IOException
{
java.net.Socket v1;
java.net.SocketAddress v2;
int v3;
io.netty.util.internal.SocketUtils$1 v0;
v0 := @this: io.netty.util.internal.SocketUtils$1;
v1 = v0.<io.netty.util.internal.SocketUtils$1: java.net.Socket val$socket>;
v2 = v0.<io.netty.util.internal.SocketUtils$1: java.net.SocketAddress val$remoteAddress>;
v3 = v0.<io.netty.util.internal.SocketUtils$1: int val$timeout>;
virtualinvoke v1.<java.net.Socket: void connect(java.net.SocketAddress,int)>(v2, v3);
return null;
}
}